### PR TITLE
Add charge voltage information

### PR DESCRIPTION
**Pull request by jonasman**
Add charge voltage information

### CODE REVIEW

File: data/ev-data.json
Comment (tijs) [inline data/ev-data.json:1]: hey man, I'm afraid I can't get my app to build with these changes:

`Failed to decode JSON from bundle due to missing key 'meta' not found`

Seems something got removed besides the addition of the voltage field. If the data would have remained in somewhat the same place I might be able to tell you where exactly but as it's just a big blob that's a bit harder to pinpoint.

Comment (jonasman) [reply data/ev-data.json:1]: I will try to make it stay in place with a different technique.

Comment (jonasman) [reply data/ev-data.json:1]: ok in the last commit, the order did not change. But github somehow cannot make the diff. I also passed the file in a json validator and it seems to be valid.

Comment (jonasman) [reply data/ev-data.json:1]: i see the problem now, spaces vs tabs

Comment (jonasman) [reply data/ev-data.json:1]: @tijs you can now review it in github, order kept and only the actual changes are shown.

Comment (tijs) [reply data/ev-data.json:1]: perfect thanks, looking good

**Comment by tijs** [comment]
@jonasman in the latest update i removed the dolphin and atto 3 from the list of 800v models as only the seal has the 800v architecture for now. also data is now exported ordered by brand, model, variant. So hopefully smaller diffs when i export from the latest master commit on.

**Pull request merged**